Patches/The Joris Experience/JorisShootCE.xml

```diff
@@ -44,6 +44,27 @@
 							</value>
 					</li>
 
+					<li Class="PatchOperationReplace">
+						<xpath>/Defs/ThingDef[defName="JE_Joku"]/verbs</xpath>

```

Comment:
This doesn't work. Honestly, I'd just leave the base attack in place and give both Joku and the BiblicalJoris better accuracy to make the vanilla attacks viable.

Reply:
Well, they are either aoe flamethrower attack or an orbital beam, not sure if accuracy benefit then much.

The range attack of bible joris is just weird, sometimes there's beam while other times there isn't.

Reply:
> Well, they are either aoe flamethrower attack or an orbital beam, not sure if accuracy benefit then much.

While they don't need to be pin-point accurate, they're currently inaccurate enough as for the creature to be often enough firing them in the the completely incorrect direction during testing. I've yet to successfully land a hit with the holy fire attack while testing it against a static pawn.

Reply:
> While they don't need to be pin-point accurate, they're currently inaccurate enough as for the creature to be often enough firing them in the the completely incorrect direction during testing. I've yet to successfully land a hit with the holy fire attack while testing it against a static pawn.

CE seems to breaks JOKU and Biblejoris' attack even when unpatched, it's pissing me off, they work fine without CE but with CE it goes in the wrong direction without any error message. 
I can't patch bible joris' shootverb as its projectile an orbital beam.